girder/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM girder/girder:latest
 
+COPY girder.local.conf /girder/girder/conf/girder.local.cfg
+
 ADD newt /girder/plugins/newt
```

Comment:
Is there a reason to mix `ADD` and `COPY` commands in this?

Reply:
no particular reason other than that one came from a copied file. I will change it